### ISSUE TITLE
[Bug]: a11y: sbb-selection-action-panel – border in high contrast are not visible

### ISSUE THREAD

**Issue by orsolya-gyovai** [sbb-design-systems/lyne-components#4395]
### Preflight Checklist

- [x] I have read the [Contributing Guidelines](https://github.com/sbb-design-systems/lyne-components/blob/main/docs/CONTRIBUTING.md) for this project.
- [x] I agree to follow the [Code of Conduct](https://github.com/sbb-design-systems/lyne-components/blob/main/docs/CODE_OF_CONDUCT.md) that this project adheres to.
- [x] I have searched the [issue tracker](https://github.com/orgs/sbb-design-systems/projects/4) for an issue that matches the one I want to file, without success.

### Bug type

Accessibility

### It affects the following packages

Components

### Which version of Lyne Components are you using

4.2.0

### Operating system

Windows 11

### Browser / Browser version

Google Chrome Version 144.0.7559.60 (Offizieller Build) (64-Bit)

### Input mode

None

### Additional settings

Windows High Contrast Mode

### What happened?

**Description**:
In Windows high contrast mode, the tiles have no border.

**Link**:
[https://lyne-storybook.app.sbb.ch/?path=/story/elements-sbb-selection-action-panel--with-radio-button-group ](url)

<img width="1930" height="702" alt="Image" src="https://github.com/user-attachments/assets/8c6741bd-e58d-471a-a54a-27aaf4dacc79" />

**Expected behavior**:
The panels should have a border in high contrast mode.

**Note**: 
The component “sbb-selection-action-panel--with-expansion-panel” is displayed correctly.

### Steps to reproduce the issue

**Windows high contrast mode**:
Settings > Accessibility > Contrast themes > Night sky

### Relevant log output

```shell

```
